pkg/storage/buckets/prefixed.go

```diff
@@ -13,54 +13,59 @@ import (
 	"storj.io/storj/pkg/storage/objects"
```

Comment:
i know this isn't about your PR, but its odd this `preefixed.go` file is under `pkg/storage/buckets/` and not in `pkg/storage/objects`

Reply:
Yeah that is unexpected. I suspect it was because Buckets contained an object store.